### PR TITLE
Added event descriptions to all expansion cards #164

### PR DESCRIPTION
**Pull request by DreyerChris**

### CODE REVIEW

File: src/locales/en/translation.json [hunk 1]
```diff
@@ -16,93 +16,114 @@
...
       "funding": {
         "name": "Funding",
-        "description": "This card must be paired with a measure card to allow the measure card to be implemented."
+        "description": "This card must be paired with a measure card to allow the measure card to be implemented.",
+        "eventDescription": "This card must be paired with a measure card to allow the measure card to be implemented."
       },
       "hazardousSubstanceRegulation": {
         "name": "Hazardous substance regulation",
-        "description": "You can pair a producer and an animal sharing the same habitat symbol to secure a habitat-tile"
+        "description": "You can pair a producer and an animal sharing the same habitat symbol to secure a habitat-tile",
```
Comment (H1D) [inline src/locales/en/translation.json:24]: BTW we should move commandBar string for this card into translation file as well.

**Review by H1D — APPROVED** [review]
Nice! Lets also have tests for `eventDescription` existence like we do have for `name` and `description` -- https://github.com/helcomsecretariat/ecosfera-baltica-digital/blob/74b9116e3ec932a2df5eb1b351917fa30c1d7daf/src/decks/__tests__/index.spec.ts

File: src/locales/en/translation.json (hunk 1 above)
Comment (DreyerChris) [reply src/locales/en/translation.json:24]: Agreed, done

**Comment by DreyerChris** [comment]
> Nice! Lets also have tests for `eventDescription` existence like we do have for `name` and `description` -- https://github.com/helcomsecretariat/ecosfera-baltica-digital/blob/74b9116e3ec932a2df5eb1b351917fa30c1d7daf/src/decks/__tests__/index.spec.ts

Thanks, done

**Pull request merged**